### ISSUE TITLE
Since v5.3.1 compile time dependencies include all maven-bundle-plugin dependencies

### ISSUE THREAD

**Issue by federicotg** [dialex/JCDP#70]
**Describe the bug**
In pom.xml maven-bundle-plugin must not be a project dependecy, just a plugin.

**To Reproduce**
1. Add JColor 5.3.1 to your project
2. Watch dozens of maven dependencies be added to your app classpath.

**Screenshots**
v5.3.0 classpath
![Captura de pantalla de 2022-05-27 14-57-51](https://user-images.githubusercontent.com/11449367/170768487-702e57bc-1161-4497-baec-b1b0b1fec897.png)

v5.3.1 classpath

![Captura de pantalla de 2022-05-27 14-58-51](https://user-images.githubusercontent.com/11449367/170768553-de2d3bac-6052-4751-875c-39024ae594a0.png)

@dialex

**Comment by federicotg** [comment]
Hi @dialex , as it stands the only workaround is adding an exclusion

        <dependency>
            <groupId>com.diogonunes</groupId>
            <artifactId>JColor</artifactId>
            <version>5.4.1</version>
            <exclusions>
                <exclusion>
                    <groupId>org.apache.felix</groupId>
                    <artifactId>maven-bundle-plugin</artifactId>
                </exclusion>
            </exclusions>
        </dependency>

**Comment by dialex** [comment]
Oh no, thanks for letting me know! That was introduced by https://github.com/dialex/JColor/pull/61

@robertcsakany the dependency you added is causing this issue.
Do you know if there's [a way](https://howtodoinjava.com/maven/maven-dependency-scopes) to tell Maven that is a "build" or "dev" only dependency, like the `<scope>test</scope>`? The goal is that people you use JColor don't have to download `maven-bundle-plugin` and its dependencies?

After reading [that link](https://howtodoinjava.com/maven/maven-dependency-scopes), feels like scope test is the way to go, as this is not needed during runtime.

**Comment by dialex** [comment]
I released the fix in `v5.5.0`, can you please confirm that it works (ie. the other dependencies are gone)?

**Comment by federicotg** [comment]
> I released the fix in `v5.5.0`, can you please confirm that it works (ie. the other dependencies are gone)?

v5.5.0 only runs in java 18+ 

maven.compiler.target should be 1.8.

@dialex 

```
 bad class file: /home/user/.m2/repository/com/diogonunes/JColor/5.5.0/JColor-5.5.0.jar(/com/diogonunes/jcolor/Ansi.class)
    class file has wrong version 62.0, should be 55.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

**Comment by dialex** [comment]
Ups, I assumed it would still be compatible with Java 8. Pushed a fix on `v5.5.1`.
